controller.py

```diff
@@ -571,7 +572,7 @@ def register(self, do_get=HTTP.get_with_timeout):
         if not url:
             message = "The public key integration document is missing an id."
             log.error(message)
-            return INVALID_INTEGRATION_DOCUMENT.detailed(message)
+            return INVALID_INTEGRATION_DOCUMENT.detailed(_(message))
```

Comment:
The _() should be on the actual text of the message so that it can be pulled out by the translation tool. So `message = _("The public key integration document is missing an id.")`